superset/initialization/__init__.py

```diff
@@ -328,18 +348,19 @@ def init_views(self) -> None:
         )
         appbuilder.add_link(
             "SQL Editor",
-            label=_("SQL Editor"),
+            label=_("SQL Lab"),
             href="/superset/sqllab/",
             category_icon="fa-flask",
             icon="fa-flask",
             category="SQL Lab",
-            category_label=__("SQL Lab"),
+            category_label=__("SQL"),
```

Comment:
is this currently using the menu permissions or the can sql lab permissions?

Reply:
it's using the can_sqllab permission